src/test/java/attendance/AttendanceBookTest.java

```diff
@@ -0,0 +1,141 @@
+        sampleAttendances.add(new Attendance(LocalDateTime.of(2024, 12, 31, 9, 45)));
+        attendanceLog = new AttendanceLog(sampleAttendances);
+        attendanceRecord = new HashMap<>();
+        attendanceRecord.put(crew, attendanceLog);
+        attendanceBook = new AttendanceBook(attendanceRecord);
+    }
+
+    @Test
+    @DisplayName("닉네임과 날짜를 입력하면 출석이 등록된다.")
+    void registerAttendanceTest() {
+        //given
+        Crew newCrew = new Crew("Lemon");
+        LocalDateTime newAttendanceDateTime = LocalDateTime.of(2024, 12, 16, 13, 30);
+
+        //when
+        Attendance attendance = attendanceBook.registerAttendance(newCrew, newAttendanceDateTime);
+
+        //then
+        assertThat(attendance.getAttendanceStatus()).isEqualTo("지각");
+        assertThat(attendance.getAttendanceDate()).isEqualTo(LocalDate.of(2024,12,16));
+        assertThat(attendance.getAttendanceTime()).isEqualTo(LocalTime.of(13,30));
+    }
+
+    @Test
+    @DisplayName("존재하지 않는 닉네임을 입력하면 예외가 발생한다")
+    void registerNonExistNameAttendanceTest() {
+        //given
+        Crew newCrew = new Crew("Murang");
+        LocalDateTime newAttendanceDateTime = LocalDateTime.of(2024, 12, 2, 13, 30);
+
+        //expected
+        assertThatThrownBy(() -> attendanceBook.registerAttendance(newCrew, newAttendanceDateTime))
+            .isExactlyInstanceOf(IllegalArgumentException.class)
+            .hasMessage("등록되지 않는 크루입니다");
+    }
+
+    @Test
+    @DisplayName("닉네임, 수정하려는 날짜, 등교 시간을 입력하여 기록을 수정할 수 있다")
+    void modifyAttendanceTest() {
+        //given
+        Crew inputCrew = new Crew("Lemon");
+        int modifyAttendanceDate = 3;
+        String modifyAttendanceTime = "10:40";
+        String format = String.format("2024-12-%02d %s", modifyAttendanceDate, modifyAttendanceTime);
+        DateTimeFormatter dfm = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+        LocalDateTime modifyDateTime = LocalDateTime.parse(format, dfm);
+
+        //when
+        List<Attendance> attendances = attendanceBook.modifyAttendance(crew, modifyDateTime);
+        Attendance oldAttendance = attendances.getFirst();
+        Attendance newAttendance = attendances.getLast();
+
+        //then
+        SoftAssertions.assertSoftly(softAssertions -> {
+            assertThat(oldAttendance.getAttendanceDate()).isEqualTo(LocalDate.of(2024,12,3));
+            assertThat(oldAttendance.getAttendanceTime()).isEqualTo(LocalTime.of(9, 50));
+            assertThat(oldAttendance.getAttendanceStatus()).isEqualTo("출석");
+
+            assertThat(newAttendance.getAttendanceTime()).isEqualTo(LocalTime.of(10, 40));
+            assertThat(newAttendance.getAttendanceStatus()).isEqualTo("결석");
+        });
+    }
+
+    @Test
+    @DisplayName("닉네임을 입력하면 이전 날까지의 출석을 확인한다")
+    void checkAttendanceRecord() {
+        //given
+        Crew inputCrew = new Crew("Lemon");
+        int lastDate = LocalDate.now().getDayOfMonth()-1;
```

Comment:
```suggestion
        int lastDate = LocalDate.now().getDayOfMonth() - 1;
```
전체적으로 Google Style Guide를 지키고 있는지 확인해볼까요? 인텔리제이의 도움을 받으면 조금 더 손쉽게 적용이 가능합니다!
[참고](https://eroul-ri.tistory.com/26)